src/components/organisms/AdminVenueView/components/Spaces/Spaces.tsx

```diff
@@ -0,0 +1,236 @@
+import React, { useCallback, useState } from "react";
+// import { updateRoom } from "api/admin";
+import {
+  faCaretRight,
+  faCaretDown,
+  faHome,
+} from "@fortawesome/free-solid-svg-icons";
+
+import { VenueTemplate, Venue_v2 } from "types/venues";
+import { RoomData_v2 } from "types/rooms";
+
+import "./Spaces.scss";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import BackgroundSelect from "pages/Admin/BackgroundSelect";
+import { VenueRoomItem } from "components/molecules/VenueRoomItem";
+import { EditSpace } from "components/molecules/EditSpace";
+import { MapPreview } from "../MapPreview";
+import { useHistory } from "react-router";
+import { Dimensions, Position } from "types/utility";
+
+interface VenueRooms {
+  text: string;
+  template?: VenueTemplate;
+  icon: string;
+}
+
+const venueRooms: VenueRooms[] = [
+  {
+    text: "Conversation Space",
+    icon: "/assets/icons/icon-room-conversation.svg",
```

Comment:
let's move the icons to `src/assets` and import them in the file

https://create-react-app.dev/docs/adding-images-fonts-and-files/

this way we get proper compile-time checks for the existence of the images.

Reply:
Fixed in: https://github.com/sparkletown/sparkle/commit/d2bc2049fe72c651d807803b6b568d74a74764f6